alpha/apps/kaltura/lib/batch2/kFlowHelper.php

```diff
@@ -2967,6 +2967,39 @@ public static function addDeliveryTagToEntry(entry $entry)
 			$entry->save();
 		}
 	}
+
+	public static function handleDelayedNotification($object)
+	{
+
+		$delayedLockedJobs = match (true)
+		{
+			$object instanceof entry => BatchJobLockPeer::retrieveByEntryIdAndStatus($object->getEntryId(), BatchJob::BATCHJOB_STATUS_DELAYED)
+		};
+
+		if (count($delayedLockedJobs) > 0)
+		{
+			foreach ($delayedLockedJobs as $job)
+			{
+				/* @var $job BatchJobLock */
+				$delayedJob = BatchJobPeer::retrieveByPK($job->getId());
+				/* @var $job BatchJob */
+				if ($delayedJob)
+				{
+					$jobData = $delayedJob->getData();
+					/* @var $jobData kEventNotificationDispatchJobData */
+					match (true)
+					{
+						$jobData->getEventDelayedConditions() == EventNotificationDelayedConditions::PENDING_ENTRY_READY =>
```

Comment:
same comment here - if something else would ever be returned (or nothing would be returned by the condition) the result will be a Fatal error (`Uncaught UnhandledMatchError: Unhandled match case true`)